controllers/controller.js

```diff
@@ -50,4 +54,53 @@ const {
     
         next(err);
       })
+  };
+
+  exports.fetchArticlesComment = (req, res, next) => {
+
+    const { article_id } = req.params;
+
+    Promise.all([
+      selectArticleById(article_id),
+      selectArticleComment(article_id)
+    ])
+
+    .then(([articleExist ,comment]) => {
+      if(!articleExist){
+       return {
+          status: 404,
+          msg: `No user found for article: ${article_id}`,
+        }
+      } else{
+      res.status(200).send({ comment });
+    }
+    })
+      .catch(err => {
+       next(err);
+      })
+  };
+
+  exports.postComment = (req, res, next) => {
+    
+    const { article_id } = req.params;
+    const newComment  = req.body;
+
+  
+     Promise.all([
```

Comment:
We don't need to worry about checking if the article exists here with a separate model call, if you try to insert a comment on an article that doesn't exist you will get a psql error :)